### PR TITLE
editorconfig

### PR DESCRIPTION
**Pull request by julianseeger**

### CODE REVIEW

**Comment by julianseeger** [comment]
@audax feel assigned ;)

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+indent_style = tab
```
Comment (audax) [inline .editorconfig]: NOOOOOOOOOO!

Comment (julianseeger) [inline .editorconfig]: erm ... it depends ^^

**Comment by julianseeger** [comment]
if we already assume spaces for everything, it get's even python inspecific...

**Pull request merged**